notebooks/integrations/openai/openai-KNN-RAG.ipynb

```diff
@@ -51,7 +51,7 @@
     "\n",
     "from getpass import getpass\n",
     "from elasticsearch import Elasticsearch, helpers\n",
-    "import wget, zipfile, pandas as pd, json, openai"
```

Comment:
is this correct?

Reply:
It was added by mistake, thanks for pointing it out. Fixed in 24b4c7e6b55a63f19c42a2f28f40fc1c6c7551ce